templates/default/backup_minecraft.sh.erb

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Should we set a path for the tar to go to?  Or do you want it to always be the script's working path?